### PR TITLE
Update TX config

### PR DESCRIPTION
**Pull request by ldodds**
Ran:

```
bundle exec rake i18n:copy_analytics_yaml
bundle exec rake i18n:generate_tx_config
```

To generate an updated config for the transifex command-line tool. The tool is used to push YAML files to transifex. 

Contents of this config has been pushed. Checking in update so we can pull down YAML when translated.

### CODE REVIEW

File: .tx/config [hunk 1]
```diff
@@ -36,38 +36,255 @@ source_lang  = en
...
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
 [o:energy-sparks:p:energy-sparks:r:mailers-mailer-yml]
 file_filter  = config/locales/<lang>/mailers/mailer.yml
 source_file  = config/locales/mailers/mailer.yml
 source_lang  = en
 type         = YML
 minimum_perc = 0
```
Comment (iant) [inline .tx/config:66]: does there need to be a config for the `config/locales/analytics` dir?  only one file in there called `common.yml`

File: .tx/config [hunk 2]
```diff
@@ -36,38 +36,255 @@ source_lang  = en
...
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
 [o:energy-sparks:p:energy-sparks:r:views-home-home-yml]
 file_filter  = config/locales/<lang>/views/home/home.yml
 source_file  = config/locales/views/home/home.yml
 source_lang  = en
 type         = YML
 minimum_perc = 0
```
Comment (iant) [inline .tx/config:192]: do we also need a config for `config/locales/views/home/privacy_and_cookie_policy.yml` here?

File: .tx/config [hunk 3]
```diff
@@ -36,38 +36,255 @@ source_lang  = en
...
 
+[o:energy-sparks:p:energy-sparks:r:views-terms_and_conditions-terms_and_conditions-yml]
+file_filter  = config/locales/<lang>/views/terms_and_conditions/terms_and_conditions.yml
+source_file  = config/locales/views/terms_and_conditions/terms_and_conditions.yml
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
```
Comment (iant) [inline .tx/config:290]: are configs needed for `activerecord.yml`, `common.yml`, `default.yml` and `devise.yml`? (maybe not as in the locale dir - but figured best to double check)

**Review by iant — APPROVED** [review]
@ldodds looks good. a few comments on possible files missing?

File: .tx/config (hunk 3 above)
Comment (ldodds) [reply .tx/config:290]: The config should have an entry for every YAML file, except anything in the `cy` directory. I've checked and these are all present, as expected.

File: .tx/config (hunk 1 above)
Comment (ldodds) [reply .tx/config:66]: You can't configure a directory, only specific files. So for now there's just the common.yml. There will be others as we extract text from the analytics.

File: .tx/config (hunk 2 above)
Comment (ldodds) [reply .tx/config:199]: Yes, this was missing. Dodgy regex in the rake task.

**Pull request merged**